core/utils.py

```
@@ -509,3 +387,62 @@ def get_jina_embedding(text: str) -> list[float] | None:
             exc_info=True,
         )
         return None
+
+
+def find_internal_link_opportunities(
+    content: str, project_pages: list, max_links: int = 10
```

Ruff (0.14.3)

Unused function argument: `max_links`

(ARG001)